run.sh

```diff
@@ -15,7 +15,8 @@ fi
 
 exec 2>&1
 
-java -jar /opt/exhibitor/exhibitor.jar \
+java -javaagent:/agents/appdynamics-jvm/javaagent.jar -Dappdynamics.agent.uniqueHostId=$(cat /agents/appdynamics-jvm/uniqueHostId) \
```

Comment:
There is no update in the Dockerfile needed, the AppDynamics java agent comes with Taupage.